src/cls/rbd/cls_rbd_types.cc

```diff
@@ -1132,6 +1132,15 @@ void TrashImageSpec::dump(Formatter *f) const {
   f->dump_unsigned("deferment_end_time", deferment_end_time);
 }
 
+void TrashImageSpec::generate_test_instances(std::list<TrashImageSpec *> &o) {
+  o.push_back(new TrashImageSpec);
+  o.push_back(new TrashImageSpec);
+  o.back()->source = TRASH_IMAGE_SOURCE_USER;
+  o.back()->name = "trash_image";
+  o.back()->deletion_time = utime_t(123, 456);
+  o.back()->deferment_end_time = utime_t(123, 456);
```

Comment:
`TrashImageSpec` has a constructor that takes these as arguments, let's use it.

Reply:
done